content/en/docs/demo/kubernetes-deployment.md

```diff
@@ -32,6 +32,10 @@ command:
 helm install my-otel-demo open-telemetry/opentelemetry-demo
 ```
 
+> **Note** The OpenTelemetry Demo Helm chart does not support being upgraded
+> from one version to another. If you need to upgrade the chart, you must first
+> delete the existing release and then install the new version.
```

Comment:
```suggestion
{{% alert title="Note" color="info" %}}

The OpenTelemetry Demo Helm chart does not support being upgraded
from one version to another. If you need to upgrade the chart, you must first
delete the existing release and then install the new version.

{{% /alert %}}
```

we prefer to make use of the docsy shortcode for alerts, since it gives us more flexibility in styling

Reply:
I updated this note, and all other notes in this file to use the docsy shortcode